### DISCUSSION TITLE
Delay on mqtt between power state and power energy

### DISCUSSION THREAD

**Discussion by DeepflashX** [arendst/Tasmota#17832] [Q&A] [answered]
Hello altogether,

with my gosund Plug with power measuring I have noticed a delay between the Power State and Power Energy values on MQTT:

![grafik](https://user-images.githubusercontent.com/117596766/215492913-8659a458-c622-49e3-8850-094fb3e40fbb.png)

The power value shows up immediately on web interface though so I don't really know why that happens.
Any ideas?

**Comment by sfromis** [comment, marked as answer]
Telemetry data is reported only every TelePeriod, default 5 minutes. Using `PowerDelta`, you can generate extra reporting when consumption level goes up or down.

**Comment by DeepflashX** [comment]
thx that helps
